### DISCUSSION TITLE
把index.php嵌入到laravel 前端报错

### DISCUSSION THREAD

**Discussion by GitHub-Ahai** [dirk1983/chatgpt#25] [Q&A]
![image](https://user-images.githubusercontent.com/56293211/223760614-d934f4d3-cc87-4322-b9f5-54f2eccd8461.png)
![image](https://user-images.githubusercontent.com/56293211/223760794-1b84b15f-1f1d-44e9-adcd-11476f7e6030.png)
把index.php嵌入到laravel 前端报错【EventSource's response has a MIME type ("text/html") that is not "text/event-stream". Aborting the connection.】我该怎么处理呢？指教下

**Comment by dirk1983** [comment]
抱歉这个我也不太懂，看看有没有其他网友能解答下吧
